cmake/modules/FindGINKGO.cmake

```
@@ -51,17 +51,19 @@ unset(GINKGO_CXX_COMPILER)
 #
 SET(_libraries "")
 FOREACH(_library ginkgo ${GINKGO_INTERFACE_LINK_LIBRARIES})
-  LIST(APPEND _libraries GINKGO_LIBRARY_${_library})
-  DEAL_II_FIND_LIBRARY(GINKGO_LIBRARY_${_library}
-    NAMES ${_library}
-    HINTS ${GINKGO_INSTALL_LIBRARY_DIR}
-    NO_DEFAULT_PATH
-    NO_CMAKE_ENVIRONMENT_PATH
-    NO_CMAKE_PATH
-    NO_SYSTEM_ENVIRONMENT_PATH
-    NO_CMAKE_SYSTEM_PATH
-    NO_CMAKE_FIND_ROOT_PATH
-    )
+  IF(_library MATCHES "ginkgo.*")
```

This looks good to me, thank you. Would you mind adding a short comment about this:
```suggestion
  # Make sure to only pick up Ginkgo's own libraries here, skipping
  # the MPI libraries that are listed here as of Ginkgo 1.5.
  IF(_library MATCHES "ginkgo.*")
```

---

Thanks Martin. Good point!